grafana/dashboards/cmode/harvest_dashboard_mcc_cluster.json

```diff
@@ -2470,8 +2908,8 @@
         "allValue": null,
         "current": {
           "selected": false,
-          "text": "stiA800mccip-20201_siteA",
-          "value": "stiA800mccip-20201_siteA"
+          "text": "stiA900mcc223224_siteA",
+          "value": "stiA900mcc223224_siteA"
```

Comment:
is this name written in json?

Reply:
i had it before in the previous checkin. so thought it is ok to have. but i have removed it. pls check again